plugins/anchor-links/index.test.js

```diff
@@ -436,6 +435,30 @@ describe('anchor-links', () => {
       ]
       `)
     })
+
+    test.skip('TODO: anchor aliases, with multiple nodes', () => {
```

Comment:
This test fails on this PR and upstream. I think we need to handle it if we want to support anchor link aliases... However, I'm also on the fence as to whether the alias functionality is worth supporting given its complexity. Fixing this case felt outside the scope of this PR.